pkg/command/one2onerestore/res.yaml

```diff
@@ -0,0 +1,36 @@
+use: 1-1-restore --cluster <id|name> --location [<dc>:]<provider>:<bucket> --snapshot-tag <tag> [flags] --source-cluster-id <id> --nodes-mapping <filepath>
+
+short: Run an ad-hoc restore of tables to a cluster that mirrors the topology (DCs, racks, nodes structure and tokens) of a backup cluster.
+
+long: |
+  This command allows you to run ad-hoc restore of tables to a cluster that mirrors the topology of a backup cluster. This means that the target cluster must have 
+  the same number of nodes in each datacenter and rack, as well as the same token assignment for each node.  
+  **Note:** This command only restores the data within the tables. 
+  You must first restore the schema of the database separately using the regular restore command with the `--restore-schema` flag.
+
+location: |
+  A list of backup locations separated by a comma, specifies places where restored backup is stored.
+  
+  The format is `[<dc>:]<provider>:<bucket>`.
+  The `<dc>` parameter is optional. It allows you to specify the datacenter whose nodes will be used to restore the data
+  from this location in a multi-dc setting, it must match Scylla nodes datacenter.
+  By default, all live nodes are used to restore data from specified locations.
+  
+  Note that specifying datacenters closest to backup locations might reduce download time of restored data.
```

Comment:
I don't think that it applies to the 1-1-restore.
Since we have node mapping, we shouldn't allow to specify the location DC.